static/js/blueslip_stacktrace.ts

```diff
@@ -0,0 +1,122 @@
+import $ from 'jquery';
+import _ from 'underscore';
+import { SourceMapConsumer, IndexedSourceMapConsumer } from 'source-map';
+import StackTraceGPS, { StackFrame } from 'stacktrace-gps';
+import ErrorStackParser from 'error-stack-parser';
+const render_blueslip_stacktrace = require('../templates/blueslip_stacktrace.hbs');
+const source_map_mappings = require('source-map/lib/mappings.wasm');
```

Comment:
Have to add a declaration
```ts
declare module "*.hbs" {
    const render: (args: { [key: string]: any }) => string;
    export = render;
}
```
otherwise typescript won't recognize it.